Runtime/Behaviours/CinemachineCollider.cs

```diff
@@ -325,7 +325,8 @@ public override float GetMaxDampTime()
                     {
                         Vector3 delta = displacement - extra.m_previousDisplacement;
                         delta = Damper.Damp(delta, damping, deltaTime);
-                        displacement = extra.m_previousDisplacement + delta;
+                        displacement = 
+                            Quaternion.Euler(state.PositionDampingBypass) * extra.m_previousDisplacement + delta;
```

Comment:
This looks fishy.  You use an unrotated prev displacement in the first line.  Shouldn't you rotate it first?